vocabularies/Analytics.xml

```diff
@@ -176,6 +179,129 @@
+        </Parameter>
+        <Parameter Name="SubtotalsAtBottom" Type="Edm.Bool" Nullable="false">
+          <Annotation Term="Core.Description" String="Whether to duplicate the group headers so that they appear before and after their descendants" />
+          <Annotation Term="Core.LongDescription">
+            <String>The entry before has [DrillState](Hierarchy.md#HierarchyType) `expanded`,
+            the entry after has DrillState `subtotal`.</String>
+          </Annotation>
+          <Annotation Term="Core.OptionalParameter">
+            <Record>
+              <PropertyValue Property="DefaultValue" String="false" />
+            </Record>
+          </Annotation>
+        </Parameter>
+        <ReturnType Type="Collection(Edm.EntityType)">
+          <Annotation Term="Core.Description" String="Output set including the instance annotation [`LevelInformation`](#LevelInformation)" />
+        </ReturnType>
+      </Function>
+      <ComplexType Name="MultiLevelExpandLevel">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Property names constituting a level in an [unnamed leveled hierarchy](#MultiLevelExpand)" />
+        <Annotation Term="Core.LongDescription">
+          <String>`DimensionProperties` must be used to identify entries in [`ExpandEntries/Entry`](#MultiLevelExpandEntry),
+          otherwise they have the same effect as `AdditionalProperties`.</String>
+        </Annotation>
+        <Property Name="DimensionProperties" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="A non-empty set of property names that define a combination of dimension values" />
+        </Property>
+        <Property Name="AdditionalProperties" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="A possibly empty set of names of additional properties of the dimensions that occur in `DimensionProperties`" />
+        </Property>
+      </ComplexType>
+      <ComplexType Name="MultiLevelExpandSiblingOrder">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Sibling order in an [unnamed leveled hierarchy](#MultiLevelExpand)" />
+        <Property Name="Property" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Property by which to sort" />
+        </Property>
+        <Property Name="Descending" Type="Edm.Boolean" Nullable="true">
+          <Annotation Term="Core.Description" String="Sort direction, ascending if not specified otherwise" />
+        </Property>
+      </ComplexType>
+      <ComplexType Name="MultiLevelExpandEntry">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Expansion state of an entry in an [unnamed leveled hierarchy](#MultiLevelExpand)" />
+        <Property Name="Entry" Type="Collection(Edm.String)" Nullable="false">
```

Comment:
This means that a value for a dimension is always of type Edm.String? There cannot be a boolean dimension etc.?

Reply:
OData does not allow `Collection(Edm.PrimitiveType)`, therefore all node IDs must be converted to strings.

Reply:
Is there a guideline how this transformation needs to happen? Would we use the same string as inside a JSON body or a URI literal or...?

Reply:
> Is there a guideline how this transformation needs to happen?

By casting the node ID to a string, in the same way as in the parameter `Hierarchy.TopLevels/ExpandLevels/NodeID`. I think that's the same as the string inside a JSON body.

Reply:
My understanding is that the node ID is always a string, but not each dimension would be a string. "casting" can mean many different things in different programming languages. I don't know how exact we want or need to be here. From my perspective, we can keep fingers crossed and see what happens :-)